granary/tests/test_reddit.py

```diff
@@ -0,0 +1,123 @@
+from oauth_dropins.webutil import testutil
+from oauth_dropins.webutil import util
+
+from granary import reddit
+
+import copy
+import json
+
+class FakeRedditor():
```

Comment:
mind adding a praw API object link and/or explanation of which API object this is faking? same below.